apple/validator.go

```diff
@@ -42,7 +42,15 @@ type Client struct {
 }
 
 // New creates a Client object
-func New() *Client {
+func New(httpClient *http.Client) *Client {
```

Comment:
Thanks for opening a PR! I do agree that we should allow a custom client and I should have added that from the start.

In the interest of not creating a breaking release, can we make a new function called NewWithClient that contains your code? That way any user of this package wont have their calls to the existing New broken.

Maybe in the next major release I will refactor it so that the default New will take in a client and a config